lib/Dredd.js

```diff
@@ -1,105 +1,141 @@
-const async = require('async');
-const parse = require('dredd-transactions/parse');
-const compile = require('dredd-transactions/compile');
-
-const configureReporters = require('./configureReporters');
-const resolveLocations = require('./resolveLocations');
-const readLocation = require('./readLocation');
-const resolveModule = require('./resolveModule');
-const logger = require('./logger');
-const TransactionRunner = require('./TransactionRunner');
-const { applyConfiguration } = require('./configuration');
-const annotationToLoggerInfo = require('./annotationToLoggerInfo');
+import async from 'async'
+import parse from 'dredd-transactions/parse'
+import compile from 'dredd-transactions/compile'
 
+import configureReporters from './configureReporters'
+import resolveLocations from './resolveLocations'
+import readLocation from './readLocation'
+import resolveModule from './resolveModule'
+import logger from './logger'
+import TransactionRunner from './TransactionRunner'
+import { applyConfiguration } from './configuration'
+import annotationToLoggerInfo from './annotationToLoggerInfo'
 
 function prefixError(error, prefix) {
-  error.message = `${prefix}: ${error.message}`;
-  return error;
+  error.message = `${prefix}: ${error.message}`
+  return error
 }
 
-
 function prefixErrors(decoratedCallback, prefix) {
   return (error, ...args) => {
-    if (error) { prefixError(error, prefix); }
-    decoratedCallback(error, ...args);
-  };
+    if (error) {
+      prefixError(error, prefix)
+    }
+    decoratedCallback(error, ...args)
+  }
 }
 
-
 function readLocations(locations, options, callback) {
-  if (typeof options === 'function') { [options, callback] = [{}, options]; }
-
-  async.map(locations, (location, next) => {
-    const decoratedNext = prefixErrors(next, `Unable to load API description document from '${location}'`);
-    readLocation(location, options, decoratedNext);
-  }, (error, contents) => {
-    if (error) { callback(error); return; }
-
-    const apiDescriptions = locations
-      .map((location, i) => ({ location, content: contents[i] }));
-    callback(null, apiDescriptions);
-  });
-}
+  if (typeof options === 'function') {
+    ;[options, callback] = [{}, options]
```

Comment:
What's the purpose of the semicolon here?

Reply:
To support initially invalid JavaScript syntax. I kept an intention to do as less changes to the existing code base as possible.